script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -1,83 +1,83 @@
 <Page1>
 ...No, I'll wait here.
-It's reckless to walk around right after coming back.
+Walking around right after coming back wouldn't help me calm down. // 帰ってきたばかりで歩き回るのはさすがに落ち着きがない
```

Comment:
Aren't these Yusuke's edits? How'd these get here

Reply:
These are my edits. No idea why they're here lol

Reply:
he probably checked out from Yusuke's branch and just has the base set to main in the pull request